dotcom-rendering/.storybook/preview-head.html

```diff
@@ -0,0 +1,149 @@
+<script>
+	document.cookie = 'gu-cmp-disabled=true';
+</script>
+
+<!-- Add Curl Loader for interactive elements-->
+<script>
+	window.curlConfig = {
+		apiName: 'require',
+	};
+	window.curl = window.curlConfig;
+</script>
+<script defer="" src="/js/curl-with-js-and-domReady.js"></script>
+
+<style>
+	@font-face {
```

Comment:
Are these comments being used?

Reply:
Following our investigation, it doesn't look like it!

The font definitions for DCR are generated here: https://github.com/guardian/dotcom-rendering/blob/7108f5dc1f9b224b099eb37bf486a8ca534d2355/dotcom-rendering/src/lib/fonts-css.ts#L282

and imported to DCR storybook here: https://github.com/guardian/dotcom-rendering/blob/26f97eecfc23a2bf658d4c223150ebb5e3cf8b4d/storybooks/.storybook/preview.js#L39

We should probably remove these font definitions from `preview-head`.

Reply:
Removed!